lib/pages/farm/components/SingleLogoFarmTitle.dart

```diff
@@ -2,20 +2,22 @@ import 'package:flutter/material.dart';
 
 import 'package:ax_dapp/service/Controller/Farms/FarmController.dart';
 import 'package:ax_dapp/pages/farm/dialogs/StakeDialog.dart';
-import 'package:ax_dapp/pages/farm/dialogs/DualStakeDialog.dart';
+//TODO: Uncomment when dualStakeDialog is functional
+// import 'package:ax_dapp/pages/farm/dialogs/DualStakeDialog.dart';
 import 'package:ax_dapp/pages/farm/modules/PageTextStyle.dart';
 import 'package:ax_dapp/pages/farm/modules/BoxDecoration.dart';
 
 Widget singleLogoFarmTitle(
     BuildContext context, bool isWeb, FarmController farm, double cardWidth) {
   //Dialog that appears when stake button is pressed
   Dialog participatingDialog;
-  if (farm.athlete == null) {
-    participatingDialog = stakeDialog(context, farm, cardWidth, isWeb);
-  } else {
-    participatingDialog =
-        dualStakeDialog(context, farm, farm.athlete!, cardWidth, isWeb);
-  }
+  //TODO: dualStakeDialog does not currently work, uncomment if else statement once it is fixed
+  // if (farm.athlete == null) {
+  participatingDialog = stakeDialog(context, farm, cardWidth, isWeb);
+  // } else {
+  //   participatingDialog =
+  //       dualStakeDialog(context, farm, farm.athlete!, cardWidth, isWeb);
+  // }
```

Comment:
Tag the ticket where this work is going to be done or I don’t wanna see these comments in the code.